test/unit/spec/media/track/remotevideotrack.js

```diff
@@ -79,8 +80,77 @@ describe('RemoteVideoTrack', () => {
       });
     });
   });
-});
 
+  describe('IntersectionObserver', () => {
+    let track;
+    let el;
+    let observeSpy;
+    let unobserveSpy;
+    let setRenderHintSpy;
+
+    before(() => {
+      global.document = global.document || new Document();
+      el = document.createElement('div');
+      setRenderHintSpy = sinon.spy();
+      track = makeTrack({ id: 'foo', sid: 'bar', kind, isSwitchedOff: false, setRenderHint: setRenderHintSpy, isEnabled: true, options: { IntersectionObserver: NullIntersectionObserver }, RemoteTrack });
+      observeSpy = sinon.spy(NullIntersectionObserver.prototype, 'observe');
+      unobserveSpy = sinon.spy(NullIntersectionObserver.prototype, 'unobserve');
+    });
+
+    after(() => {
+      observeSpy.restore();
+      unobserveSpy.restore();
+      if (global.document instanceof Document && kind === 'video') {
+        delete global.document;
+      }
+    });
+
+    context('when an element is attached', () => {
+      beforeEach(() => {
+        track.attach(el);
+      });
+
+      it('IntersectionObserver observe is called', () => {
+        sinon.assert.callCount(observeSpy, 1);
+      });
+
+      it('_setRenderHint gets called with { enable: true }', () => {
+        sinon.assert.calledWith(setRenderHintSpy, { enabled: true, renderDimensions: { height: undefined, width: undefined } });
```

Comment:
can we create an element with some height and width to that we can verify it here instead of them being `undefined`